### PR TITLE
Change README.md

### PR DESCRIPTION
**Pull request by sankalpsp07**
Changes suggested by @HarshKapadia2

### CODE REVIEW

**Comment by HarshKapadia2** [comment]
Merge this one, @SirusCodes 😛 😂

**Review by SirusCodes — CHANGES_REQUESTED** [review]
The changes which you have made is not true and we won't accept spreading false information for our users.

Please make proper changes to the docs.

**Comment by HarshKapadia2** [comment]
What has been stated is the truth and nothing but the truth.
It is shameful to see the owner of this repository trying to suppress the truth. 😛

**Comment by SirusCodes** [comment]
> What has been stated is the truth and nothing but the truth.
> It is shameful to see the owner of this repository trying to suppress the truth. 😛

Defaming someone in the name of truth is not which is expected from you...

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-test++
+Darshan is G.O.A.T.
```
Comment (SirusCodes) [inline README.md:1]: This looks perfect.

**Review by SirusCodes — APPROVED** [review]

**Comment by HarshKapadia2** [comment]
LMAO 😂

**Comment by SirusCodes** [comment]
😂😂😂 Power of OS

**Pull request merged**